src/ResourceManagement/Storage/Microsoft.Azure.Management.Storage/generate.cmd

```diff
@@ -4,7 +4,7 @@
 ::
 
 @echo off
-set autoRestVersion=0.16.0-Nightly20160406
+set autoRestVersion=1.0.0-Nightly20161122
```

Comment:
This should be updated to the latest (201612001)  this will make type declarations more readable by removing the usage of fully qualified type names